README.md

```diff
@@ -7,6 +7,8 @@ A project based learning activity for people who are getting started with Git an
 1. Press the space bar to begin.
 2. Use the up and down arrow keys to rotate the shape.
 3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
```

Comment:
Is this clear enough?  By "no empty spaces" do you mean empty spots or holes in the row?